src/pods/toolbar/components/export-button/export-button.component.tsx

```diff
@@ -34,14 +50,15 @@ export const ExportButton = () => {
       />
     );
 
-    downloadSvg(svg);
+    downloadImage(svg, viewBoxSize);
```

Comment:
Please do not remove, I want to have both options, just commenting, uncommenting one to enable on or the other.

Reply:
ok, checked is ok